Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -103,6 +103,8 @@ vtkMRMLMarkupsNode::~vtkMRMLMarkupsNode()
 
   if (this->Measurements)
     {
+    this->Measurements->RemoveAllItems(); // Trigger modified event to unobserve measurements
```

Comment:
Do we still need this line? Usually we want to avoid firing modified events in destructors because it can cause a cascade of calls that may assume that this object is still valid while in fact it may be already partially destructed.

Reply:
There is no leak on exit when deleted so I think it's not needed. I removed it.